src/components/data-exploration/differential-expression-tool/DiffExprResults.jsx

```diff
@@ -220,7 +232,11 @@ const DiffExprResults = (props) => {
       />
       {
         pathwayAnalysisModal && (
-          <LaunchPathwayAnalysisModal onCancel={() => setPathwayAnalysisModal(false)} />
+          <LaunchPathwayAnalysisModal
+            onApplyFilters={applyAdvancedFilters}
+            onCancel={() => setPathwayAnalysisModal(false)}
+            advancedFiltersAdded={advancedFilters?.length}
```

Comment:
If we want to use this as a bool just pass a bool: `advancedFilters.length > 0`

Also in which case can `advancedFilters` be `undefined` or `null` to justify using the `.?`? 
I would set an initialState with `[]`originally so that isn't a possibility.